### DISCUSSION TITLE
Show repository visibility status in an account’s repository list

### DISCUSSION THREAD

**Discussion by ghost** [community/community#15460] [Mobile]
On the GitHub website, repositories display a visible tag when viewing yours or others' repositories list.

On mobile, it does not display these tags. This, for example, makes it hard to distinguish which of my own repositories are public or private.

**Comment by Gil8010** [comment]
@@![2021-05-16.jpg](https://user-images.githubusercontent.com/96861637/165009269-ea805687-8501-46c2-989d-db0c6dd06cec.jpg)

**Reply by ghost** [reply]
very rad lol

does the dog approve of my feedback?
